### DISCUSSION TITLE
Question: Is there a way to *receive* content as well?

### DISCUSSION THREAD

**Discussion by derwaldgeist** [videokit-ai/natshare#108] [Q&A] [answered]
Hi Yusuf, thanks for this great asset. Is there a way to also *receive* content instead of sharing it with other apps? My use case: I want to let the user take a picture with the device camera and then hand it over to my app. If NatShare cannot do this, are you aware of any other asset that can?

**Comment by olokobayusuf** [comment, marked as answer]
Hey @derwaldgeist , NatShare doesn't support this. I think implementing it will be somewhat complicated, as your app (natively) has to register for what forms of media it can receive, and has to handle actually receiving those items. I'm not aware of any asset that does this. I'll add it to the NatShare wishlist.

**Reply by derwaldgeist** [reply]
Thanks for the super-fast response, highly appreciated! For me, photos and videos would be the assets I'd like to receive, but even if only photos were possible, this would already be awesome. Happy Holidays!

**Reply by derwaldgeist** [reply]
PS: Wasn't aware that Github now also features discussions. This is cool!

**Reply by olokobayusuf** [reply]
Ikr, this is such a great feature to build community. I plan to use it all the way. I'll look into this feature as it would be a nice addition to the API. In the past I've been resistant to making the scope of NatShare any larger than it is, but I'm gradually shifting to creating a full ecosystem with the entire NatSuite Framework.

Happy holidays!

**Reply by derwaldgeist** [reply]
This would be a great addition. Happy holidays!
